packages/zowe-explorer/package.json

```diff
@@ -1652,6 +1652,12 @@
           },
           "description": "%Zowe-Environment%",
           "scope": "window"
+        },
+        "zowe.disableTemporaryDownloadFolderCleanup": {
+          "type": "boolean",
+          "default": false,
+          "description": "%zowe.disableTemporaryDownloadFolderCleanup%",
```

Comment:
@JillieBeanSim Would it be less confusing for users if we labeled this setting in the positive? Saying "Disable Temporary Downloads Folder Cleanup" forces the user to think in double negatives when setting this to `false`, i.e. "I am not disabling `x`, so therefore I am enabling `x`".

Perhaps we could use one of the following:
- "Enable Temporary Downloads Folder Cleanup" (true/false)
- "Temporary Downloads Folder Cleanup"
  - Settings description: "Enable temporary downloads folder cleanup" (true/false).
  
I remember we had a similar discussion about double negatives when we added the global enable/disable profile validation setting, and this is what we ended up going with:

<img width="377" alt="Screen Shot 2021-06-29 at 9 34 06 AM" src="https://user-images.githubusercontent.com/45975633/123806769-400dd900-d8bd-11eb-977b-2c2892e4223c.png">

Reply:
@lauren-li so we don't want to include enable or disable in the label, just an open ended label, and add Allow to the description like with validation?

Reply:
`zowe.files.temporaryDownloadsFolderCleanup`: `Allow deletion of downloaded file content at editor deactivation (closing a Window) to support using separate VS Code windows editing the same files.`

Reply:
![image](https://user-images.githubusercontent.com/49491949/123811622-6897d200-d8c1-11eb-9526-ba8a99749a3d.png)

Reply:
Sorry @lauren-li , the above examples are all wrong.

What we will need to do if we follow the validation example is change to this with default true and allow the user to disable it by unchecking the box.
![image](https://user-images.githubusercontent.com/49491949/123817059-d3e3a300-d8c5-11eb-88ab-b7b73e35fc92.png)

Reply:
@JillieBeanSim I would rephrase the setting description to:
```
Allow deletion of downloaded file content during deactivation (closing a window). Disable this setting to support using separate VS Code windows editing the same files.
```
Do we want to add a note that disabling the setting means the temporary files will persist after the window is closed?

Actually, would it be clearer if we renamed the setting to `Persist temporary downloads`? (The default would have to be set to `false` again, though.)